bread/views/edit.py

```diff
@@ -24,8 +25,8 @@ class EditView(
     """TODO: documentation"""
 
     accept_global_perms = True
-    fields = None
-    urlparams = (("pk", int),)
+    fields: Optional[List[str]] = None
```

Comment:
Fields can also be hg elements, this is helpfull to customize single fields without having to overwrite the whole layout method.
```fields: Optional[List[Union[hg.BaseElement, str]]] = None```